bee-node/Dockerfile

```diff
@@ -0,0 +1,58 @@
+############################
+# Prepare dependencies recipe
+############################
+FROM lukemathwalker/cargo-chef as planner
+
+WORKDIR /src
+COPY . .
+
+RUN cargo chef prepare --recipe-path recipe.json
+
+############################
+# Dependency cache
+############################
+FROM lukemathwalker/cargo-chef as cacher
+
+RUN apt-get update && \
+    apt-get install cmake clang -y
+
+WORKDIR /src
+COPY --from=planner /src/recipe.json recipe.json
+
+RUN cargo chef cook --release --recipe-path recipe.json
+
+############################
+# Build
+############################
+FROM rust:1 as build
+
+WORKDIR /src
+COPY . .
```

Comment:
Same as above. Are we sure that `.` at source wouldn't falsly point to `./src`, which does not exist?

Reply:
Could you also please add a `.dockerignore` file, which filters out any build artifacts residing in `/target`?